tests/acceptance/features/coreApiSharees/sharees.feature

```diff
@@ -22,11 +22,11 @@ Feature: search sharees
     And the HTTP status code should be "200"
     And the "exact users" sharees returned should be empty
     And the "users" sharees returned should be
-      | Sharee One | 0 | sharee1 |
+      | Sharee One | 0 | sharee1 | sharee1 |
```

Comment:
why extra `sharee1`?

Reply:
it's `shareWithAdditionalInfo` value. if `OCIS_SHOW_USER_EMAIL_IN_RESULTS=true` -> `shareWithAdditionalInfo=userEmail` else `shareWithAdditionalInfo=userName`

https://github.com/owncloud/ocis/pull/8861/files#diff-e2ace41e6f6027457c14cbea8d6c1a8116c1870e022c3d941a79c8b2513a6adfR186-R191

I know it's not very explicit, but I didn't change the code much because this will get rid of the ocs endpoint soon. It's more relevant for stable-5.0